Products/ZenModel/migrate/fixConsumerAndQueryCreds.py

```diff
@@ -0,0 +1,60 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+
+sm.require("1.0.0")
+
+creds = """zenossCredentials:
+       username: "{{(getContext . "global.conf.zauth-username")}}"
+       password: "{{(getContext . "global.conf.zauth-password")}}"
+"""
+
+
+class FixConsumerQueryCreds(Migrate.Step):
+    """Fix the credentials for consumer and query services"""
+
+    version = Migrate.Version(108, 0, 0)
```

Comment:
The version specified in the migration scripts references the database version, not the code version.  The database version should be consistent between the different release trains (i.e. dev, 5.2.x, 5.1.x, etc.)